src/providers/proxy/proxy_id.c

```diff
@@ -1488,6 +1488,152 @@ static int get_initgr(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+static int compare_group_members(struct proxy_id_ctx *ctx,
```

Comment:
This functions actually checks if `db_groupname` is in `gids`, right?
I.e. this has nothing to do with 'members' and thus function name is misleading...
Could you please add a short free text comment to this and `remove_group_members()` functions (what is the purpose)?

Reply:
I've changed both functions purposes and names. What do you think about them? I'd prefer to avoid writing a description if a good name is enough.